magento/pwa-studio
# [feature] Checkout Page Price Adjustments

## Description

This PR uses the Cart page adjustments features (minus shipping estimator). Infact without the styling changes to the PR that is the only thing to point out - there is a Checkout Page specific Price Adjustments component since the Cart page adjustments component had the estimator.

So the major changes to this PR are styling updates. You can find reference to the specific changes in the JIRA issue.

## Related Issue





Closes [PWA-248](https://jira.corp.magento.com/browse/PWA-248).

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Add item to cart and go to `/cart` page. Go through price adjustments entering coupon code, gift card, and selecting some gift options. Changes to the price should update the summary.
2. Proceed to Checkout and then fill out the form to get to the third step, Payment Information.
3. Before clicking "review order", verify that the adjustments sections have carried over information from the cart page.
4. Go to local storage and delete `apollo-cache-persist` and refresh the checkout page.
5. Proceed through the form but don't click review order yet.
6. Enter adjustments like you did on the cart page. They should apply and update the price summary.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
